### PR TITLE
[hsa-ext-rocr] fix 404 on deb file

### LINKED ISSUE #291: [hsa-ext-rocr] deb 404 not found
Downloading hsa-ext-rocr-dev_1.1.30501.0-rocm-rel-3.5-34-def83d8_amd64.deb...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found

The available .deb at 2020 july 4th is:
https://repo.radeon.com/rocm/apt/debian/pool/main/h/hsa-ext-rocr-dev/hsa-ext-rocr-dev_1.1.30500.0-rocm-rel-3.5-30-def83d8_amd64.deb

### PR DESCRIPTION
**Pull request by keyvank**
Closes #291

### CODE REVIEW

File: hsa-ext-rocr/PKGBUILD [hunk 1]
```diff
@@ -3,16 +3,16 @@
 # Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
 
 pkgname=hsa-ext-rocr
-pkgver=3.5.1
+pkgver=3.5.0
```
Comment (acxz) [inline hsa-ext-rocr/PKGBUILD:6]: The AUR will not accept an update to the PKGBUILD where the pkgver has decreased. I suggest for now we keep the pkgver at 3.5.1 and bump the pkgrel by 1 even as we change to the working url. While technically incorrect, AMD will release a new version soon enough.

Comment (keyvank) [reply hsa-ext-rocr/PKGBUILD]: Reasonable, done! @acxz

**Comment by acxz** [comment]
Also another weird thing on PKGBUILD line 19.
We need to hardcode `3.5.0` where `$pkgver` is. Ugly I know, but this is temporary.

**Comment by keyvank** [comment]
> Also another weird thing on PKGBUILD line 19.
> We need to hardcode `3.5.0` where `$pkgver` is. Ugly I know, but this is temporary.

Both of them?

**Comment by acxz** [comment]
Yeah for consistency that would be best.

**Comment by acxz** [comment]
Sweet, @keyvank thanks for iterating. LGTM!

**Pull request merged**